### PR TITLE
KEYCLOAK-15641: Update Required Actions

### PR DESCRIPTION
**Pull request by nebelsprachen**
* Update Required Actions
* Split up the monolith into procedure and concept modules
* Create subassembly
* https://issues.redhat.com/browse/KEYCLOAK-15641

### CODE REVIEW

File: server_admin/topics/users/proc-setting-default-required-actions.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[id="proc-setting-default-required-actions_{context}"]
+= Default Required Actions
+
+You can also specify required actions that will be added to an account in two ways.
```
Comment (andymunro) [inline server_admin/topics/users/proc-setting-default-required-actions.adoc]: How about:  "You can use two methods to specify required actions that will be added to an account."

File: server_admin/topics/users/proc-setting-default-required-actions.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* When a new user is created, use the *Add User* button on the user list screen
+* Use the <<_user-registration, user registration>> link on the login page.  
+
+
+You can specify the default required actions 
+
+.Procedure
+. Click *Authentication* in the left menu.
+. Click the `Required Actions` tab.
+. Click the checkbox in the `Default Action` column of the required actions that you want to be executed when a brand new user logs in.
```
Comment (andymunro) [inline server_admin/topics/users/proc-setting-default-required-actions.adoc]: How about "Click the checkbox in the `Default Action` column for one or more required actions. When a new user first logs in, those actions will be executed."

File: server_admin/topics/users/proc-setting-default-required-actions.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[id="proc-setting-default-required-actions_{context}"]
+= Default Required Actions
+
+You can also specify required actions that will be added to an account in two ways.  
+* When a new user is created, use the *Add User* button on the user list screen
+* Use the <<_user-registration, user registration>> link on the login page.  
+
+
+You can specify the default required actions
```
Comment (andymunro) [inline server_admin/topics/users/proc-setting-default-required-actions.adoc]: I think this sentence can be removed.  You already introduced it well.

File: server_admin/topics/users/proc-setting-required-actions.adoc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[id="proc-setting-required-action_{context}"]
+== Setting Required Action
+
+.Procedure
+. Navigate to the _Required User Actions_ list box
```
Comment (andymunro) [inline server_admin/topics/users/proc-setting-required-actions.adoc]: Period at the end.

**Comment by nebelsprachen** [comment]
@andymunro Please review, when you have a chance :)

**Comment by andymunro** [comment]
Thanks so much, Joan, but I see a Resolve Conflicts.  I think it's easy to fix.

**Pull request merged**